tests/report_test.py

```diff
@@ -100,4 +100,4 @@ def _r(fc, sim_name, analysis_model, shared_model=None):
     r = fc.sr_run_sim(data, analysis_model)
     if shared_model:
         r = fc.sr_run_sim(data, shared_model, timeout=2, forceRun=False)
-        pkunit.pkeq("completed", r.state)
+    pkunit.pkeq("completed", r.state)
```

Comment:
This seemed like a bug to me

Reply:
Yes, and you can remove shared_model since it doesn't seem to be used. (Remove `__future__` as well).

Reply:
e0899d07129b3ccbb1eac4eb836a98b3c623d32f